docs/BreakingChanges/018 BlockingCollectionT.TryTakeFromAny does not throw anymore.md

```diff
@@ -1,4 +1,4 @@
-## 18: BlockingCollection<T>.TryTakeFromAny doesn't throw anymore
+## 18: BlockingCollection<T>.TryTakeFromAny does not throw anymore
```

Comment:
While we're fixing this one up, may want to do something about the \<T\> not showing up in the title. Maybe just escape the <> with \\.